core/station/api/spec.did

```diff
@@ -1567,6 +1603,17 @@ type ChangeCanisterResourceAction = variant {
   Create;
 };
 
+// The target canister of a request to change a managed canister.
+type ChangeManagedCanisterResourceTarget = variant {
+  Any;
+  Canister : principal;
+};
+
+// The actions that are available for managed canister changes.
+type ChangeManagedCanisterResourceAction = variant {
+  Create : ChangeManagedCanisterResourceTarget;
```

Comment:
Creation does not combine with the Canister ID specifiers since there's no canister ID yet. Calls etc. are an option though and I could add them here. Alternatively, we could introduce a new request type for calls.

Reply:
Sounds good. Let me do it tomorrow morning.

Reply:
Done!